notebooks/community/sdk/sdk_private_endpoint.ipynb

```diff
@@ -0,0 +1,1015 @@
+{
```

Comment:
<p>load_ext tensorboard -&gt; if tensorboard not used in notebook, you should remove the extension</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
Fixed, removed the extension